### DISCUSSION TITLE
Bug in displaying ipynb content correctly

### DISCUSSION THREAD

**Discussion by laggykiller** [community/community#60710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a ipynb with the following line:

`!sed -i '1i#include <string.h>' src/rwpng.c`

In github, it is displayed incorrectly as:

`!sed -i '1i#include ' src/rwpng.c`

Google Colab displays correctly.

You can see this bug appearing in https://github.com/laggykiller/sticker-convert/blob/6a9a344d9704bfe2865cbdaa08ee881e1963d5db/sticker_convert_colab.ipynb

This is my first post here. Please let me know if I have opened the issue in wrong location, or need more info, thank you.

**Comment by look** [comment]
Thanks for the report, I've filed an issue with the team.
